server/src/interfaces/asset.interface.ts

```diff
@@ -133,7 +133,7 @@ export interface MetadataSearchOptions {
   numResults: number;
 }
 
-export type AssetPathEntity = Pick<AssetEntity, 'id' | 'originalPath' | 'isOffline'>;
+export type AssetPathEntity = Pick<AssetEntity, 'id' | 'originalPath' | 'isOffline' | 'isExternal'>;
```

Comment:
You should revert this.

Reply:
Thanks for catching that. I have reverted this change.